tests/test_stringify.py

```diff
@@ -0,0 +1,52 @@
+def test_stringify_func__empty_bracket():
```

Comment:
This for writing tests for this new function! 👍

Reply:
thanks!